### ISSUE TITLE
Final Summer Presentation on 8/3

### ISSUE THREAD

**Issue by ananya-gangadhar** [AguaClara/UASB#92]
A list of tasks that need to be taken care of before the UASB Final presentation on **3 August 2018 at 11:15 AM**:

- [ ] Add tapioca test slides to presentation
- [ ] Decide whether to keep info about general wastewater treatment on slides
- [ ] Add speaker notes
- [ ] Designate speakers for each section
- [ ] Practise presentation, if needed
- [ ] Address all of Juan's comments on the report
- [ ] Get rid of previous comments from report
- [ ] Add tapioca test video/image to report
- [ ] Add tipping bucket frame design video to report and fabrication manual
- [ ] Create doctests in the report if time permits 
- [ ] Have a tea party to destress, then get back to work!

@isakaminsky @IanCullings  @AlyciaStorch

**Comment by IanCullings** [comment]
@ananya-gangadhar we decided to just start from scratch on a new template.  We sketched out the topics for us to discuss [on the presentation here](https://docs.google.com/presentation/d/1hZF6zNaL1DjUnw9Y7dfBTTy_UKkT6FQ3-bpJ3-LKSso/edit#slide=id.g3de4d76f23_0_0), feel free to make any edits you would like and discuss them here.  Tomorrow we'll work on this together and practice a few times.

The report doesn't need to be completed till the 10th, although the more we can get this done the better.  I would move all additions for the report to that issue (#88)
